network/base/models.py

```diff
@@ -232,17 +232,18 @@ def has_submitted_data(self):
     # observaton has at least 1 payload that has been verified good
     @property
     def has_verified_data(self):
-        return self.data_set.filter(vetted_status='verified').count()
+        return self.data_set.filter(vetted_status=u"('verified', 'Verified')").count()
```

Comment:
These filters were not working unless I used the choices tuple in a unicode quote!